tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectImportUtil.java

```diff
@@ -500,6 +500,63 @@ else if( tempRecord.isTheme() )
         return isValid;
     }
 
+    public static IStatus validateSDKPath(final String currentPath)
+    {
+        IStatus retVal = Status.OK_STATUS;
+
+        if( !org.eclipse.core.runtime.Path.EMPTY.isValidPath( currentPath ) )
+        {
+            retVal = ProjectCore.createErrorStatus( "\"" + currentPath + "\" is not a valid path." );
+        }
+        else
+        {
+            IPath osPath = org.eclipse.core.runtime.Path.fromOSString( currentPath );
+
+            if( !osPath.toFile().isAbsolute() )
+            {
+                retVal = ProjectCore.createErrorStatus( "\"" + currentPath + "\" is not an absolute path." );
```

Comment:
the validateSDKPath method just validate the sdk itself path and it won't validate the app.server.parent.dir path.  The sdk validate mthod mainly check the sdk internal properties. Do you mean we just call the SDKUtil.isValidSDKLocation method to verify sdk location is validation?